server/modules/spotifyPlaylist.js

```diff
@@ -0,0 +1,81 @@
+var SpotifyWebApi = require('spotify-web-api-node');
+
+var scopes = ['playlist-read-private', 'playlist-modify-public', 'playlist-modify-private'],
+  clientId = "873379a32a354918b0a54f4f79736716",
+  clientSecret = process.env.CLIENT_SECRET,
+  state = 'jazz-and-eri-women-powahhh';
```

Comment:
optional, to make sure you're not getting gaked callbacks. Ideally it would be something random or dynamic based on which user is making the request.